src/scripts/main.js

```diff
@@ -21,7 +21,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    const listItem = document.createElement('li');
+
+    listItem.textContent = key;
+
+    if (Object.keys(data[key]).length > 1) {
+      createTree(tree, data[key]);
```

Comment:
The recursive call to `createTree` should use `listItem` as the first argument instead of `tree`. This ensures that the nested list is appended to the correct parent list item, maintaining the correct hierarchy.